src/errors/LatestVersionUnavailableError.ts

```diff
@@ -1,7 +1,13 @@
 export class LatestVersionUnavailableError extends Error {
-  constructor(public packageName: string, public customRegistry?: string) {
+  constructor(
+    public packageName: string,
+    public customRegistry?: string,
+    public isPatch?: boolean
```

Comment:
I think `isPatchWildCard` or `isWildCardPatch` would be more descriptive.

Reply:
[2b59a31](https://github.com/standardhealth/fhir-package-loader/pull/15/commits/2b59a31682443905280984131c2923b69a86b706)